### PR TITLE
Firefox iOS 9.0 String Export (Dry-Run Do Not Merge - Xcode 8.3.3) - Almost Looking Good

### PR DESCRIPTION
**Pull request by st3fan**

### CODE REVIEW

File: af/firefox-ios.xliff [hunk 1]
```diff
@@ -485,31 +485,24 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
     </header>
     <body>
+      <trans-unit id="A modern mobile browser from Mozilla, the non-profit committed to a free and open web.">
```
Comment (st3fan) [inline af/firefox-ios.xliff]: These IDs are not correct. They should not be the strings, but instead be actual identifiers.

File: af/firefox-ios.xliff [hunk 2]
```diff
@@ -1067,16 +1057,15 @@
         <target>Besoek werf in elk geval</target>
         <note>Button label to temporarily continue to the site from the certificate error page</note>
       </trans-unit>
+      <trans-unit id="ExternalLink.ConfirmMessage">
```
Comment (st3fan) [inline af/firefox-ios.xliff]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1390952

File: af/firefox-ios.xliff [hunk 3]
```diff
@@ -1755,6 +1816,10 @@
         <target>Serif</target>
         <note>Font type setting in the reading view settings</note>
       </trans-unit>
+      <trans-unit id="Setting_OfferClipboardBar_status">
```
Comment (st3fan) [inline af/firefox-ios.xliff]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1390947

File: af/firefox-ios.xliff [hunk 4]
```diff
@@ -1926,26 +2015,67 @@
...
+      <trans-unit id="Settings.TrackingProtectionOption.OnLabel">
+        <source>Always On</source>
+        <note>Settings option to specify that Tracking Protection is always on.</note>
       </trans-unit>
       <trans-unit id="Settings.TroubleShootSync.Title">
         <source>Troubleshoot</source>
         <target>Probleemoplossing</target>
         <note>Title of link to help page to find out how to solve Sync issues</note>
       </trans-unit>
+      <trans-unit id="Settings_OfferClipboardBar_title">
```
Comment (st3fan) [inline af/firefox-ios.xliff]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1390947

File: af/firefox-ios.xliff (hunk 1 above)
Comment (st3fan) [inline af/firefox-ios.xliff]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1390953

File: templates/firefox-ios.xliff [hunk 5]
```diff
@@ -1476,6 +1548,10 @@
         <source>Serif</source>
         <note>Font type setting in the reading view settings</note>
       </trans-unit>
+      <trans-unit id="Setting_OfferClipboardBar_status">
+        <source>When opening Firefox</source>
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: How is this string used?

File: templates/firefox-ios.xliff [hunk 6]
```diff
@@ -2270,7 +2414,7 @@
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName">
-        <source>Share To</source>
+        <source>$(MOZ_BUNDLE_DISPLAY_NAME)</source>
```
Comment (flodolo) [inline templates/firefox-ios.xliff]: Is this expected?

Comment (st3fan) [inline templates/firefox-ios.xliff]: Yes, the Share To extension should be named 'Firefox' or 'Firefox Beta' - it should have the same name as on the Home Screen - so not translated.

File: templates/firefox-ios.xliff (hunk 5 above)
Comment (st3fan) [inline templates/firefox-ios.xliff]: This is a new setting.  I'll add a screenshot to the string comment.

**Pull request closed without merging**